File: Respostas/Bee_1049/1049.c
#include <stdio.h>

int main() {
	char animal_type[20], classe[20], feed[20];
	
	scanf("%s %s %s", animal_type, classe, feed);
	
	if (animal_type[0] == 'v' && classe[0] == 'a' && feed[0] == 'c') printf("aguia\n");
	if (animal_type[0] == 'v' && classe[0] == 'a' && feed[0] == 'o') printf("pomba\n");
	
	if (animal_type[0] == 'v' && classe[0] == 'm' && feed[0] == 'o') printf("homem\n");
	if (animal_type[0] == 'v' && classe[0] == 'm' && feed[0] == 'h') printf("vaca\n");
	
	if (animal_type[0] == 'i' && classe[0] == 'i' && feed[2] == 'm') printf("pulga\n");
	if (animal_type[0] == 'i' && classe[0] == 'i' && feed[2] == 'r') printf("lagarta\n");
	
	if (animal_type[0] == 'i' && classe[0] == 'a' && feed[0] == 'h') printf("sanguessuga\n");
	if (animal_type[0] == 'i' && classe[0] == 'a' && feed[0] == 'o') printf("minhoca\n");
	
	return 0;
}

File: Respostas/Bee_1049/1049.java
import java.io.IOException;
import java.util.Scanner;

public class Main {
	public static void main(String[] args) throws IOException {
		String animal_type, classe, feed;
		
		Scanner input = new Scanner(System.in);
		
		animal_type = input.nextLine();
		classe = input.nextLine();
		feed = input.nextLine();
		
		if (animal_type.charAt(0) == 'v' && classe.charAt(0) == 'a' && feed.charAt(0) == 'c')
			System.out.println("aguia");
		if (animal_type.charAt(0) == 'v' && classe.charAt(0) == 'a' && feed.charAt(0) == 'o')
			System.out.println("pomba");
		
		if (animal_type.charAt(0) == 'v' && classe.charAt(0) == 'm' && feed.charAt(0) == 'o')
			System.out.println("homem");
		if (animal_type.charAt(0) == 'v' && classe.charAt(0) == 'm' && feed.charAt(0) == 'h')
			System.out.println("vaca");
		
		if (animal_type.charAt(0) == 'i' && classe.charAt(0) == 'i' && feed.charAt(2) == 'm')
			System.out.println("pulga");
		if (animal_type.charAt(0) == 'i' && classe.charAt(0) == 'i' && feed.charAt(2) == 'r')
			System.out.println("lagarta");
		
		if (animal_type.charAt(0) == 'i' && classe.charAt(0) == 'a' && feed.charAt(0) == 'h')
			System.out.println("sanguessuga");
		if (animal_type.charAt(0) == 'i' && classe.charAt(0) == 'a' && feed.charAt(0) == 'o')
			System.out.println("minhoca");
	}
}